_episodes/l1-02-committing-history.md

```diff
@@ -53,7 +53,7 @@ what changes. The `--global` part of the command sets this information for
 any projects on which you might work on this computer. Therefore you only need
 to perform the above commands once for each new computer Git is installed on.
 
-[setup instructions]: {% link setup.md %}
+[setup instructions]: {{ page.root }}{% link setup.md %}
```

Comment:
In the intermediate course we use the pre-existing `[lesson-setup]` link that is already included in the `links.md` (also in [this course](https://github.com/ImperialCollegeLondon/introductory_grad_school_git_course/blob/gh-pages/_includes/links.md?plain=1#L28)).

We do it [this way](https://github.com/ImperialCollegeLondon/intermediate_grad_school_git_course/blob/gh-pages/_episodes/00-course_overview.md?plain=1#L23). Meaning that you would replace `[setup-instructions]` above with `[setup instructions][lesson-setup]` and remove this line entirely.

```suggestion
```

Reply:
Cool, yeah that makes sense. I've essentially copied what you linked to in the intermediate course now.